docs/Features.rst

```diff
@@ -92,6 +96,7 @@ with your social intranet in a familiar environment.
    :width: 10.00000cm
 .. |image4| image:: images/mobile/documents_share.png
 .. |image5| image:: images/mobile/mobile_upload_files.png
+.. |image6| image:: images/mobile/commentAS_mobile.png
```

Comment:
The file commentAS_mobile.png is missing

Reply:
I added it right now, thanks